### ISSUE TITLE
About TAB module

### ISSUE THREAD

**Issue by JKBox** [CryhanFang/CLIP2Video#8]
TAB module looks similar with [T2VLAD](https://arxiv.org/abs/2104.10054)

**Comment by FangmingZhou** [comment]
That is actually the same thing, you can also find it in some more previous work, such as Collaborative Experts

**Comment by FangmingZhou** [comment]
> That is actually the same thing, you can also find it in some more previous work, such as Collaborative Experts

T2VLAD in CVPR2021, too

**Comment by fly-dragon211** [comment]
Yes, and the diagram is similar with [T2VLAD](https://arxiv.org/abs/2104.10054).

**Comment by FangmingZhou** [comment]
T2Vlad even not been mentioned by this work, what a surprise...
